### ISSUE TITLE
Problem with migration (doctrine:migrations:diff not work!)

### ISSUE THREAD

**Issue by mitalcoi** [Sylius/Sylius#8537]
Step to reproduce:
1)Install fresh Sylius app
2)apply migrations (or run ``` sylius:install  ```) 
3)try to generate diff  (``` doctrine:migrations:diff ```)
A lot of sql will be generated by doctrine. If you apply it and reproduce step 1-3 situation will happen again

System info:
php71
10.2.8-MariaDB
Sylius: ^1.0@beta

**Comment by pjedrzejewski** [comment]
Should be fixed in #8536.

**Comment by alekseyp** [comment]
I think I've got the same issue, this is due to how MariaDB returns _'NULL'_ instead of _NULL_.
Doctrine Issue: https://github.com/doctrine/doctrine2/issues/6565

To fix, just comment _server_version_ in you config file

```
diff --git a/app/config/config.yml b/app/config/config.yml
index ef5947e..6f1dd62 100644
--- a/app/config/config.yml
+++ b/app/config/config.yml
@@ -37,7 +37,7 @@ doctrine:
         dbname: '%database_name%'
         user: '%database_user%'
         password: '%database_password%'
-        server_version: '5.5'
+        #server_version: '5.7'
         charset: UTF8
 
 doctrine_migrations:
```
